libraries/provider_firewall_rule_iptables.rb

```diff
@@ -92,7 +122,15 @@ def build_firewall_rule(type = nil)
         if [:pre, :post].include?(new_resource.direction)
           firewall_rule << '-t nat '
         end
+
+        # Iptables order of prameters is important here see example output below:
```

Comment:
It seems the output order is determined by the input order, so I'm not really sure this is necessary. If it is, we should write a test enforces it & demonstrates that it is required (test will fail when we change order).